backend/models/issueModel.js

```diff
@@ -0,0 +1,37 @@
+import mongoose from "mongoose";
+
+const medicalEmergencySchema = new mongoose.Schema(
+  {
+    title: String,
+    overview: {},
+    treatment: {},
+    content: {},
+  },
+  {
+    methods: {
+      getName() {
```

Comment:
What is the intended purpose of the getName method in the schema? Since mostly we will only be fetching complete documents, not sure if this is necessary.

Reply:
also, do we need to add optional flag to the content field?

Reply:
Thank you for your message. I made a commit removing the getName method, which was included before by way of habit. Yes, this would be unnecessary for the purpose of fetching complete documents. As tested and discussed, fields are optional by default.